db/migrate/20230415180602_create_fun_questions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFunQuestions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :fun_questions do |t|
+      t.references :user, null: true, foreign_key: true
+      t.references :time_period, null: true, foreign_key: true
+      t.text :question_body
```

Comment:
Unless the requirements state that these questions must be more than 255 characters it would be better to use strings instead of text.